src/styles/_services.scss

```diff
@@ -2,7 +2,119 @@
   #hero {
     height: 100vh;
     min-height: 700px;
-    background-image: url("/assets/img/services/finsecptx-service-banner-2000.jpg");
+    background-image: url("/assets/img/services/hero-phone-boot-bg-2000.jpg");
+    background-position: center;
+    background-size: cover;
```

Comment:
I had a crack at combining those elements regarding background into the same line, but it didn't work for me, so I've kept it how it is for the moment. I read through the whole section and tried everything I could think of 🎅